python/cugraph-pyg/cugraph_pyg/examples/gcn_dist_mnmg.py

```diff
@@ -59,10 +59,6 @@ def init_pytorch_worker(global_rank, local_rank, world_size, cugraph_id):
 
     cupy.cuda.set_allocator(rmm_cupy_allocator)
 
-    from cugraph.testing.mg_utils import enable_spilling
-
-    enable_spilling()
-
```

Comment:
Sorry, I missed that initially, but it appears we've enabled spilling in every example. Could we remove all instances of it? LGTM otherwise

Reply:
Done